### PR TITLE
Implementing toolbar and add posts button changes

### PR DESCRIPTION
**Pull request by crcommons**
This pull request makes the following changes:
-implements styling changes for toolbar dropdown and "See New Posts" button. Updates pattern library version.

Testing checklist:
In Data View:

- [x] Change class of "See New Posts" button to ng-show from ng-hide (or wait until new posts come in. Confirm it looks nice. 
- [x] Should be below "Bulk Actions"
- [x] should be fixed so that it stays at top when scrolling (known bug appears once you scroll to bottom. Already filed)

- [x] Open bulk actions. confirm that 'X' button is in top right corner and that it stays in position as you resize browser
- [x] confirm alignment of buttons and text in toolbar stays nice as you resize browser.

Fixes ushahidi/platform# .

Ping @ushahidi/platform

### CODE REVIEW

**Comment by rowasc** [comment]
Looks great. One tiny thing, not even sure if it is a bug.When I scroll, the "x of x posts"  message is no longer there, and there's a white space instead. Is that by design/intentional? 

screen shot 2017-10-13 at 5 14 56 pm
screen shot 2017-10-13 at 5 14 53 pm

**Comment by crcommons** [comment]
@rowasc That's the intended design. The header with X of X reports and Bulk actions aren't meant to be fixed/sticky. so as you scroll down, you will see the button and margins but not those other things.

File: app/main/posts/common/collection-toggle/collection-toggle-button.html [hunk 1]
```diff
@@ -2,5 +2,5 @@
     <svg class="iconic">
         <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="/img/iconic-sprite.svg#grid-three-up"></use>
     </svg>
-    <span class="button-label" translate="set.add_to_collection">Add to collection</span>
+    <span class="button-label hidden" translate="set.add_to_collection">Add to collection</span>
```
Comment (rowasc) [inline app/main/posts/common/collection-toggle/collection-toggle-button.html]: I'm not sure about this. Why is this being hidden ?

**Comment by rowasc** [comment]
@crcommons ok awesome then. Left a question because I don't really understand that bit

File: app/main/posts/common/collection-toggle/collection-toggle-button.html (hunk 1 above)
Comment (crcommons) [reply app/main/posts/common/collection-toggle/collection-toggle-button.html]: Seth had me add it for accessibility reasons.

**Pull request merged**